ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -498,7 +498,10 @@ public void SaveAs(String file, SaveOptions options)
             else if (_loadSource == XLLoadSource.File)
             {
                 if (String.Compare(_originalFile.Trim(), file.Trim(), true) != 0)
+                {
                     File.Copy(_originalFile, file, true);
+                    File.SetAttributes(file, FileAttributes.Normal);
```

Comment:
And if `_loadSource` is something else? I think the `File.SetAttributes` should be lower down, outside the `if` statements.

Reply:
I'm wrong. It's not needed for the other `_loadSource` values. I do, however, think we need to catch that specific exception and rethrow it with a more meaningful message. What do you think?

Reply:
That sounds reasonable. On the other hand, File.Copy may fail due to plenty of reasons (invalid path, network failure, disk is full, user has no permissions, file is busy, etc.). Each of them would cause a different exception, more or less specific.
Does this make sense to process some of these cases (like overwriting a read-only file) separately, making the exception more specific than is thrown by the framework? Probably, yes, assuming than user would expect some kind of IO.Exception to be thrown on saving failure, but I am not sure.